src/Mono.Android/map.csv

```diff
@@ -71,19 +49,6 @@
 0,,,android/view/Surface.FX_SURFACE_MASK,98304
 0,,,android/view/Surface.FX_SURFACE_NORMAL,0
 
-18,Android.Hardware.Location.GeofenceTransition,Entered,android/hardware/location/GeofenceHardware.GEOFENCE_ENTERED,1
```

Comment:
Why did this "block" of mappings need to be moved, or otherwise touched? It makes for a noisier diff.

Reply:
Those lines below around line # 100 were **manually** added because they could not be automatically generated (because, they didn't exist in DroidDoc anymore!).

Now we generate enum const list based on api-*.xml.in they can be automatically generated again and therefore they went below # 100. It is a good thing!